curl/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ if (BUILDING_LEATHERMAN)
 endif()
 
 add_leatherman_library(src/client.cc src/request.cc src/response.cc)
+if (WIN32)
+    symbol_exports(leatherman_curl "${CMAKE_CURRENT_LIST_DIR}/inc/leatherman/curl/export.h")
```

Comment:
As an option? That could be convenient, sure. I ran into trouble doing it on other platforms though, so I should probably investigate that more first.